test/tests.yaml

```diff
@@ -11,10 +11,12 @@
 #     - denied-boot: stubby refused to boot kernel.
 #     - warned-cmdline: stubby warned that secureboot would not allow.
 tests:
-  - name: "sb-shim-allowed"
-    runtime: "root=atomix console=ttyS0"
+  - name: "sb-shim-allowed-built-in-no-rt-no-marker"
```

Comment:
Clearly this is a test we need - but it should be added to, rather than replacing, the one that only has a runtime and has empty built-in, right?

Reply:
You're right, I'll add it back in.  I wasn't thinking about no-builtin, runtime only which we check.